src/main/java/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,21 @@
 package dto;
 
 import java.math.BigDecimal;
+import javax.validation.constraints.Min;
 import lombok.Data;
+import lombok.NonNull;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NonNull
     private String title;
+    @NonNull
     private String author;
+    @NonNull
     private String isbn;
+    @NonNull
+    @Min(0)
```

Comment:
You use annotations of lombok, and don't use annotations from package of validation.